docs/content.zh/release-notes/flink-1.14.md

```diff
@@ -0,0 +1,424 @@
+---
+title: "Release Notes - Flink 1.14"
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Notes - Flink 1.14
+
+These release notes discuss important aspects, such as configuration, behavior, or dependencies,
+that changed between Flink 1.13 and Flink 1.14. Please read these notes carefully if you are
+planning to upgrade your Flink version to 1.14.
+
+### DataStream API
+
+#### Expose a consistent GlobalDataExchangeMode
+
+##### [FLINK-23402](https://issues.apache.org/jira/browse/FLINK-23402)
+
+The default DataStream API shuffle mode for batch executions has been changed to blocking exchanges
+for all edges of the stream graph. A new option `execution.batch-shuffle-mode` allows to change it
+to pipelined behavior if necessary.
+
+#### Allow @TypeInfo annotation on POJO field declarations
+
+##### [FLINK-12141](https://issues.apache.org/jira/browse/FLINK-12141)
+
+`@TypeInfo` annotations can now also be used on POJO fields which, for example, can help to define
+custom serializers for third-party classes that can otherwise not be annotated themselves.
```

Comment:
Swap topics. Let's start with the simple one and then the more technical, advanced later.